Source/CMakeLists.txt

```diff
@@ -49,6 +49,18 @@ set(WOOF_SOURCES
     m_swap.h
     midifile.c midifile.h
     mmus2mid.c mmus2mid.h
+    net_client.c net_client.h
```

Comment:
Are all the other `net_*` files taken pristine from Choco or are there any changes that need to get documented?

Reply:
The only changes are for other games and for fixing gcc/clang warnings.

Reply:
Also, I commented out "SecureDemo" experiment.

Reply:
Never heard of that. 🤷

Reply:
> 

Look at `NET_StartSecureDemo` in `net_query.c` Not sure how it works.

Reply:
It doesn't seem to be actually used anywhere.